src/components/app.tsx

```diff
@@ -10,8 +10,43 @@ import {EncodingPane} from './encoding-pane';
 import {Header} from './header';
 import {ViewPane} from './view-pane';
 
+import {
+  ActionHandler,
+  DatasetAsyncAction,
+  datasetUrlLoad,
+} from '../actions';
+
+
+
+interface DataURL {
+  name: string;
+  url: string;
+};
+export type VoyagerData = DataURL | Object[];
+export type VoyagerConfig = Object;
```

Comment:
Sure. At the moment its just empty since we don't know whats going into it (though stuff like disabling the dataset selector or not would be there)